comparison.md

```diff
@@ -209,7 +209,7 @@ Help improve our documentation! If you find anything wrong or which could be imp
 				<td><strong>External Caching Systems (CDNs, Fastly)</strong></td>
 				<td>✅ Varnish automatically detects external Caching system and disables caching</td>
 				<td>✅</td>
-				<td>✅</td>
+				<td>✅ External caches respect Varnish/Drupal headers, Drupal modules available</td>
```

Comment:
ok, guess there are two ways on how to handle a CDN, in our case we had issues where the client purges the external cache only. But then the files did not update, as the CDN just called our Varnish, which still had the old version. So this created a lot of confusion, why there are now two things you have to purge.